dataset/pretrain_datasets.py

```diff
@@ -210,7 +208,8 @@ def __getitem__(self, index):
             else:
                 # ssv2 & ava & other rawframe dataset
                 if 'SomethingV2' in video_name:
-                    self.skip_length = self.ssv2_skip_length
+                    self.new_step = 2
+                    self.skip_length = self.new_length * self.new_step
```

Comment:
I think we need to add code to line 191 to restore self.new_step to its original value.